src/commands/exit.ts

```diff
@@ -0,0 +1,10 @@
+import { ExitExecuteResult } from "../result.ts";
+
+export function exitCommand(args: string[]): ExitExecuteResult {
+  const code = parseInt(args[0], 10);
+  if (isNaN(code)) throw new Error("invalid result code");
```

Comment:
Instead of throwing, we should probably log this message to stderr (`, stderr: ShellPipeWriter`) then return the same exit code that the exit command does in this scenario (I think `1`, but my wsl is broken at the moment so I can't easily check).

Reply:
Looks like `2`:

```
#!/bin/bash
exit aksjdksaj
```
```
$ bash ./t.sh
./t.sh: line 2: exit: aksjdksaj: numeric argument required
```
```
$ printf '%d\n' $?
2
```